enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -120,7 +118,21 @@ <h1>Linked learners</h1>
       {% endfor %}
       </tbody>
     </table>
-
+    {% if learners.has_other_pages %}
```

Comment:
Where this is coming from?

Reply:
learners is a Page object now, so it has `has_other_pages` property.